moveit_core/planning_request_adapter/include/moveit/planning_request_adapter/planning_request_adapter.h

```diff
@@ -63,6 +63,15 @@ class PlanningRequestAdapter
   {
   }
 
+  /// Initialize parameters using the passed NodeHandle
+  // TODO - Make initialize() a pure virtual function
+  virtual void initialize(const ros::NodeHandle& node_handle)
+  {
+    ROS_WARN_NAMED("planning_request_adapter",
```

Comment:
What about using `typeid(*this).name()` to provide a useful hint which adapter is effected.